src/common/errors.c

```diff
@@ -0,0 +1,7 @@
+#include "errors.h"
+
+
+const error_t errors[] =
+{
+     {"Bed Heating Failed", "Heating failed,\nplease check the\nbed heater and\nthermistor wiring\nfor possible damage."}
```

Comment:
The error texts should not contain any forced formatting like manual text wrapping. 

However, I understand that it might be currently needed because of broken text wrapping in GUI. Something, we should fix asap...

Reply:
použito z důvodu (pravděpodobně) nesprávné funkcionality pro WINDOW_CLS_TEXT